src/applications/claims-status/reducers/serialize.js

```diff
@@ -25,27 +29,49 @@ const getDocsAssociatedWithTrackedItems = docs =>
   docs.filter(isAssociatedWithAnyTrackedItem);
 
 const getDocsAssociatedWithTrackedItem = (item, docs) => {
-  const predicate = isAssociatedWithTrackedItem(item);
-  return docs.filter(predicate);
+  return docs.filter(isAssociatedWithTrackedItem(item));
 };
 
 const associateDocsWithTrackedItems = (items, docs) => {
-  return items.map(item => {
-    const newItem = { ...item };
-    const associatedDocs = getDocsAssociatedWithTrackedItem(newItem, docs);
-    newItem.documents = associatedDocs || [];
-    newItem.date = getTrackedItemDate(item);
-    return newItem;
-  });
+  return items.map(item => ({
+    ...item,
+    documents: getDocsAssociatedWithTrackedItem(item, docs),
+  }));
 };
 
 const filterAssociatedDocs = docs =>
   docs.filter(doc => !isAssociatedWithAnyTrackedItem(doc));
 
+const transformUnassociatedDocs = docs =>
+  docs.map(doc => ({
+    ...doc,
+    date: doc.uploadDate,
+  }));
+
+const transformAssociatedTrackedItems = items =>
+  items.map(item => ({
+    ...item,
+    // <tempfix>
```

Comment:
Do we have an established pattern I should use here for marking code as temporary? Ideally this would be some sort of consistent token we can search for during clean-up work.